NEWS.md

```diff
@@ -0,0 +1,276 @@
+# News
```

Comment:
'News' sounds more like world politics, can we please use a more suitable term, e.g. migration guides, upgrades, changes, changelog?